firestore-send-email/functions/src/index.ts

```diff
@@ -351,17 +355,8 @@ async function processWrite(
 
       const payload = snapshot.data();
 
-      // We expect the payload to contain a message object describing the email
-      // to be sent. If it doesn't, we can't do anything.
-      if (typeof payload.message !== "object") {
-        logs.invalidMessage(payload.message);
-        return false;
-      }
-
-      // The record has most likely just been created by a client, so we need to
-      // initialize the delivery state.
-
```

Comment:
Do we need to keep the comments here? As in they should follow the code and to remain for the code below?